docs/framework/docker/aspnetmvc.md

```diff
@@ -50,33 +45,31 @@ The finished application is located in the
 
 ## Prerequisites
 
-At a minimum, your development machine must be running
+The development machine must be running
 [Windows 10 Anniversary Update](https://www.microsoft.com/en-us/software-download/windows10/)
 or [Windows Server 2016](https://www.microsoft.com/en-us/cloud-platform/windows-server). 
 
-Before starting, you need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/),
+[Docker for Windows](https://docs.docker.com/docker-for-windows/) must be installed,
 version 1.12 Beta 26, or newer. Windows Container support is only available
 in the Beta channel at this time.
 
 > [!IMPORTANT]
-> If you are using Windows Server 2016, you need to follow the
-> instructions for [Container Host Deployment - Windows Server](https://msdn.microsoft.com/virtualization/windowscontainers/deployment/deployment)
-> before you can run Docker containers.
+> If you are using Windows Server 2016, follow the
+> instructions for [Container Host Deployment - Windows Server](https://msdn.microsoft.com/virtualization/windowscontainers/deployment/deployment).
 
-After installing and starting Docker, you'll need to right-click on the
-tray icon and select **Switch to Windows containers** in order to run
+After installing and starting Docker,  right-click on the
+tray icon and select **Switch to Windows containers**. This is required to run
 Docker images based on Windows. This command takes a few seconds to
 execute:
 
 ![Windows Container][windows-container]
 
 ## Publish script
 
-The first step is to get all the assets that you'll need to load into
-a Docker image in one place. Fortunately, you can use the Visual Studio
-**Publish** command to create a publish profile for your application. This
-profile will put all the assets in one directory tree that you will
-copy to your target image later in this tutorial.
+Collect all the assets that you need to load into
```

Comment:
To me it sounds confusing compared to the original.  "Collect all..." sounds like an imperative to do something I don't yet know how to do.